src/Internal/Service/PaymentProcessingService.php

```diff
@@ -7,9 +7,21 @@
 
 namespace WCPay\Internal\Service;
 
+use Exception;
```

Comment:
Note to self: We might want to introduce a different exception class for the payment process. It's alright for now.

Reply:
Definitely! Good point, added a comment besides it 42a3a4ceea33b619bd9a5cb844e532b0208bf41b